docs/user/03-11-customize_secret.md

```diff
@@ -0,0 +1,62 @@
+# Customize the Default Credentials and Access
+
+> [!NOTE]
+> This feature is still being developed and will be available soon.
+
+You can customize the `sap-btp-manager` Secret and manage your own default configuration of the SAP BTP Operator module.
+
+## Context
+
+When you create SAP BTP, Kyma runtime, the `sap-btp-manager` Secret is automatically created as the default Secret managing the SAP BTP Operator's resources. 
+Because of Kyma's automatic reconciliation, any changes to the Secret are reverted, and the previous settings are restored within up to 24 hours.
+See [Preconfigured Credentials and Access](03-10-preconfigured-secret.md#credentials).
+
+To customize the `sap-btp-manager` Secret and prevent your changes from being reverted, you must stop the Secret's reconciliation.
+With the customized Secret, you can perform the following actions:
+
+* Manage your own default configuration of SAP BTP Operator
+* Migrate the service instances you created outside of the Kyma environment to your Kyma cluster
+
+## Procedure
+
+To customize the `sap-btp-manager` Secret, perform the following steps:
+
+* Label the Secret with `kyma-project.io/skip-reconciliation: 'true'`.
+* Provide the following credentials from your SAP Service Manager instance: **clientid**, **clientsecret**, **sm_url**, and **tokenurl**.
+* Optionally, provide your **cluster_id**. Otherwise, it is generated automatically.
+* Optionally, add the **credentials_namespace** parameter and provide the name of your custom namespace for Secrets with credentials to communicate with the SAP Service Manager.  
```

Comment:
After reading this sentence I understood that the custom namespace should already contain credentials to use. Is that correct understanding? If not, could you change to something like "...the name of your custom namespace where  sap-btp-operator-service Secret will be generated."?

Reply:
That's correct, we assume the namespace already exists and it could contain anything user wants there. `sap-btp-service-operator` Secret also exists at this point, we just move it to the other namespace selected by user, it should remain the default Secret for communication with SAP Service Manager.

Reply:
This is a rule. Using "will" is against the [content guidelines](https://github.com/kyma-project/community/blob/main/docs/guidelines/content-guidelines/02-style-and-terminology.md#tenses).

Reply:
The **credentials_namespace** is the namespace for secrets, it does not concern only the `sap-btp-service-operator` secret. The sentence `...the name of your custom namespace where sap-btp-operator-service Secret will be generated."` does not hold the true meaning of the namespace. User can create `sap-btp-manager` secret with the setting already provided and there won't be any movement of the secrets.